packages/aws-client/tests/S3/test-multipartCopyObject.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+/*
```

Comment:
I suggest to remove the comments here.  It's not related to the tests.
Add a test for `moveObject` with 0 size file